core/dbt/contracts/graph/unparsed.py

```diff
@@ -234,6 +241,26 @@ def __post_init__(self) -> None:
 
         self.deprecation_date = normalize_date(self.deprecation_date)
 
+        if self.time_spine:
+            columns = (
+                self.get_columns_for_version(self.latest_version)
+                if self.latest_version
+                else self.columns
+            )
+            column_names_to_columns = {column.name: column for column in columns}
+            if self.time_spine.standard_granularity_column not in column_names_to_columns:
+                raise ParsingError(
+                    f"Time spine standard granularity column must be defined on the model. Got invalid "
```

Comment:
would be great to include the model name in the error message here! Same for the ParsingError a few lines down in this method

Reply:
Updated!